.prettierrc

```diff
@@ -0,0 +1,10 @@
+{
+  "printWidth": 120,
+  "tabWidth": 2,
+  "useTabs": false,
+  "semi": true,
+  "singleQuote": true,
+  "trailingComma": "es5",
+  "arrowParens": "avoid",
+  "bracketSpacing": false
```

Comment:
personal pref, but could you set "bracketSpacing" to `true` please?

Reply:
also my preference, i changed it and reformatted the code :-)